source/user-manual/reference/ossec-conf/localfile.rst

```diff
@@ -595,6 +597,96 @@ For example, we may want to read a Python Traceback output as one single log, re
       <multiline_regex replace="wspace">^Traceback</multiline_regex>
    </localfile>
 
+ignore
+^^^^^^^^^^^^^^^
+
+This allows you to configure a pcre2 expression to ignore specific log lines. Multiple expressions can be configured by repeating the ignore label. The log will be ignored as soon as one of the expressions matches, OR behavior.
+
++--------------------+---------------------------------------------------------------+
+| **Default Value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any `regex <regex.html#regex-os-regex-syntax>`_,              |
+|                    | `sregex <regex.html#sregex-os-match-syntax>`_ or              |
+|                    | `pcre2 <regex.html#pcre2-syntax>`_ expression.                |
++--------------------+---------------------------------------------------------------+
+
+The attribute type is optional.
+
++-------------+-----------------------------------------+-------------+---------------+
+| Attribute   |              Description                | Value range | Default value |
++=============+=========================================+=============+===============+
+| **type**    | Allows to set regular expression type   |   osregex   |     pcre2     |
+|             |                                         +-------------+               |
+|             |                                         |   osmatch   |               |
+|             |                                         +-------------+               |
+|             |                                         |   pcre2     |               |
++-------------+-----------------------------------------+-------------+---------------+
+
+For example, to ignore events related to configuration changes in the audit log:
+
+.. code-block:: xml
+
+  <localfile>
+      <log_format>audit</log_format>
+      <location>/var/log/audit/audit.log</location>
+      <ignore type="osregex">type=\.+_CHANGE</ignore>
```

Comment:
Change the type of this `ignore` to show that types can be different in the same `localfile`.